Experiment dir : Checkpoints/I3DWTrans-NvGesture-M-20240516-172933
05/16 05:29:33 PM gpu device = 0
05/16 05:29:33 PM Model:I3DWTrans, Total Categories:25
pretrained:  /home/minjae/ws/MotionRGBD/Checkpoints/model_best-Nv-M.pth.tar-v1
Missing keys:
 ['dtn.mlp_head_small.1.weight', 'dtn.mlp_head_small.1.bias', 'dtn.mlp_head_media.1.weight', 'dtn.mlp_head_media.1.bias', 'dtn.mlp_head_large.1.weight', 'dtn.mlp_head_large.1.bias']
phase:  train
learnig rate:  0.005
05/16 05:29:35 PM Dataset:NvGesture, Modality:rgb
05/16 05:29:35 PM Training Data Size is: 1336
05/16 05:29:35 PM Average Train Data frames are: 80, max frames: 80, min frames: 80
05/16 05:29:35 PM Dataset:NvGesture, Modality:rgb
05/16 05:29:35 PM Validation Data Size is: 538
05/16 05:29:35 PM Average Train Data frames are: 80, max frames: 80, min frames: 80
Namespace(config='config/NvGesture_init.yml', eval_only=False, local_rank=0, nprocs=1, save_grid_image=False, save_output=False, demo_dir='./demo', resume='', distill_lamdb=0.0, drop_path_prob=0.5, save='Checkpoints/I3DWTrans-NvGesture-M-20240516-172933', seed=123, data='/home/minjae/ws/data/nvGesture/Video_data/rgb', splits='/home/minjae/ws/data/nvGesture/Video_data', visdom={'enable': False, 'visname': 'NvGesture'}, dataset='NvGesture', batch_size=2, test_batch_size=2, num_workers=4, learning_rate=0.005, learning_rate_min=1e-05, momentum=0.9, weight_decay=0.0003, init_epochs=0, epochs=100, report_freq=100, optim='SGD', dist=False, vis_feature=True, DEBUG=False, scheduler={'name': 'cosin', 'patience': 4, 'warm_up_epochs': 3}, loss={'name': 'CE', 'labelsmooth': True}, MultiLoss=True, loss_lamdb=[1, 0.5, 0.5, 0.5], distill=1.0, Network='I3DWTrans', pretrained='/home/minjae/ws/MotionRGBD/Checkpoints/model_best-Nv-M.pth.tar-v1', resumelr=True, sample_duration=64, sample_size=224, grad_clip=5.0, SYNC_BN=1, w=4, temper=0.4, recoupling=True, knn_attention=0.7, sharpness=True, temp=[0.04, 0.07], frp=True, SEHeads=1, N=6, type='M', flip=0.0, rotated=0.5, angle='(-10, 10)', Blur=False, resize='(256, 256)', crop_size=224, low_frames=16, media_frames=32, high_frames=48, num_classes=25, phase='valid')
05/16 05:29:35 PM param size = 38.142957MB
Traceback (most recent call last):
  File "/home/minjae/ws/MotionRGBD/tools/train.py", line 500, in <module>
    main(args.local_rank, args.nprocs, args)
  File "/home/minjae/ws/MotionRGBD/tools/train.py", line 164, in main
    train_acc, train_obj, meter_dict_train = train(train_queue, model, criterion, optimizer, epoch, local_rank)
  File "/home/minjae/ws/MotionRGBD/tools/train.py", line 328, in train
    for step, (inputs, heatmap, target, _) in enumerate(train_queue):
  File "/home/minjae/anaconda3/envs/gesture/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 634, in __next__
    data = self._next_data()
  File "/home/minjae/anaconda3/envs/gesture/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1346, in _next_data
    return self._process_data(data)
  File "/home/minjae/anaconda3/envs/gesture/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1372, in _process_data
    data.reraise()
  File "/home/minjae/anaconda3/envs/gesture/lib/python3.9/site-packages/torch/_utils.py", line 644, in reraise
    raise exception
FileNotFoundError: Caught FileNotFoundError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/home/minjae/ws/MotionRGBD/lib/datasets/base.py", line 190, in Sample_Image
    ori_image = Image.open(self.get_path(imgs_path, a))
  File "/home/minjae/anaconda3/envs/gesture/lib/python3.9/site-packages/PIL/Image.py", line 3247, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/home/minjae/ws/data/nvGesture/Video_data/rgb/rgb/class_03/subject3_r0/000000.jpg'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/minjae/anaconda3/envs/gesture/lib/python3.9/site-packages/torch/utils/data/_utils/worker.py", line 308, in _worker_loop
    data = fetcher.fetch(index)
  File "/home/minjae/anaconda3/envs/gesture/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/minjae/anaconda3/envs/gesture/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/minjae/ws/MotionRGBD/lib/datasets/NvGesture.py", line 43, in __getitem__
    self.clip, skgmaparr = self.image_propose(self.data_path, sl)
  File "/home/minjae/ws/MotionRGBD/lib/datasets/base.py", line 240, in image_propose
    return Sample_Image(data_path, sl)
  File "/home/minjae/ws/MotionRGBD/lib/datasets/base.py", line 192, in Sample_Image
    ori_image = Image.open(os.path.join(imgs_path, "MDepth-%08d.png" % int(a+1))) # For NTU fusion
  File "/home/minjae/anaconda3/envs/gesture/lib/python3.9/site-packages/PIL/Image.py", line 3247, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/home/minjae/ws/data/nvGesture/Video_data/rgb/rgb/class_03/subject3_r0/MDepth-00000001.png'
None
